example_experiments/asreader_johannes_dataset.json

```diff
@@ -0,0 +1,25 @@
+{
+    "model_class": "AttentionSumReader",
+    "model_serialization_prefix": "models/multiple_choice_qa/asreader_johannes",
+    "preferred_backend": "theano",
+    "encoder": {
```

Comment:
This file is now out of date, with the merge of the #175.  You need a "default" key in here.